pandas/tests/test_base.py

```diff
@@ -418,7 +418,7 @@ def test_value_counts_unique_nunique_null(self, null_obj):
                     values = o._shallow_copy(v)
                 else:
                     o = o.copy()
-                    o[0:2] = iNaT
```

Comment:
what would happen to the arrange if you tried to assign iNaT? raise? coerce to object? do we have a test for that?

Reply:
coerce to object, and yes